yamlExamples/1/example.yaml

```diff
@@ -0,0 +1,42 @@
+# same as defined in current version
```

Comment:
Not really sure what this comment means, and it's the same in both files

Reply:
This means that agent will be defined in the yaml file as it is defined now.